molecule/default/molecule-no-systemd.yml

```
@@ -23,25 +23,40 @@ platforms:
   # now.
   # - name: amazonlinux2
   #   image: amazonlinux:2
```

Even though this is commented out, we still want it to be updated with the new stuff...just in case it is ever uncommented:
```suggestion
  # - image: amazonlinux:2
  #   name: amazonlinux2
  #   platform: amd64
```